examples/btc-to-evm-fungible-transfer/src/transfer.ts

```diff
@@ -0,0 +1,139 @@
+import dotenv from 'dotenv';
+import { createBitcoinFungibleTransfer } from '@buildwithsygma/btc';
+import { ECPairFactory, ECPairAPI } from 'ecpair';
+import {
+  initEccLib,
+  networks,
+  payments,
+  Psbt
+} from "bitcoinjs-lib";
+import * as tinysecp from 'tiny-secp256k1';
+import { broadcastTransaction, calculateFee, getFeeEstimates, getTweakedSigner, toXOnly } from './utils';
+
+dotenv.config();
+
+const DESTINATION_ADDRESS = process.env.DESTINATION_ADDRESS;
+const DESTINATION_DOMAIN_ID = Number(process.env.DESTINATION_DOMAIN_ID);
+const BLOCKSTREAM_URL = process.env.BLOCKSTREAM_URL;
+const PRIVATE_KEY = process.env.PRIVATE_KEY;
+const RESOURCE_ID = process.env.RESOURCE_ID;
+const SOURCE_DOMAIN_CAIPID = process.env.SOURCE_DOMAIN_CAIPID
+const FEE_ADDRESS = 'tb1p0r2w3ugreaggd7nakw2wd04up6rl8k0cce8eetxwmhnrelgqx87s4zdkd7'
+const FEE_AMOUNT = 1000000;
+const EXPLORER_URL = process.env.EXPLORER_URL;
+
+if (!DESTINATION_ADDRESS || !PRIVATE_KEY || !DESTINATION_DOMAIN_ID || !BLOCKSTREAM_URL || !RESOURCE_ID || !SOURCE_DOMAIN_CAIPID) {
+  throw new Error('Please provided needed env variavles in .env file');
+}
+
+type InputData = {
+  hash: string;
+  index: number;
+  witnessUtxo: { value: number; script: Buffer };
+  tapInternalKey: Buffer;
+};
+
+async function btcToEvmTransfer(): Promise<void> {
+  // pre setup
+  const testnet = networks.testnet;
+  initEccLib(tinysecp as any);
+  const ECPair: ECPairAPI = ECPairFactory(tinysecp);
+
+  console.log('Transfer BTC to EVM');
+
+  // tweaking signer
+  const { tweakedSigner, publicKey } = getTweakedSigner(ECPair, tinysecp, testnet, PRIVATE_KEY!);
+
+  // Generate an address from the tweaked public key
+  const p2pktr = payments.p2tr({
+    pubkey: toXOnly(tweakedSigner.publicKey),
+    network: testnet,
+  });
+
+  const p2pktrAdddress = p2pktr.address as string;
+  // address here should match the one that you generated for your private key
+  console.log('Taproot address to use', p2pktrAdddress)
+
+  const feeEstimatesPerBlockConfirmation = await getFeeEstimates(BLOCKSTREAM_URL!);
```

Comment:
It's probably better to trow an error with the message that BLOCKSTREAM_URL is doesn't set